quiz.js

```diff
@@ -7,26 +7,22 @@ let quizTime = 0;
 let numWrong = 0;
 let numCorrect = 0;
 
-for (i = 0; i < 3; i++) {
+for (let i = 0; i < questions.length; i++) {
```

Comment:
It's good practice to declare your variable before assigning them. You can run into unexpected behavior if you don't declare first (with `let` or `const`).

Using `questions.length` here instead of the hard coded `3` is a bit more robust if you add more questions in the future.